packages/server/src/semester/setCourse.command.ts

```diff
@@ -1,104 +0,0 @@
-import { Command, Positional } from 'nestjs-command';
-import { Injectable } from '@nestjs/common';
-import { Season } from '@koh/common';
-import { SemesterService } from './semester.service';
-import { SemesterModel } from './semester.entity';
-
-@Injectable()
-export class SetSemesterCommand {
-  constructor(private readonly semService: SemesterService) {}
-  @Command({
-    command: 'semester:toggleActiveSemester  <mode> <semester> <year>',
-    describe: '(disable or enable) all the classes in a given semester',
```

Comment:
Wait this seems like a useful feature?

Unless i'm mistaken and it doesn't actually work

Reply:
This shouldn't really be used in production + we have seed controller. So I don't think we need a command.